foo/bar.js

```diff
@@ -0,0 +1,24 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    if (term == "chicken") {
+	return true;
+    }
+    if (term == "poulet") {
+        return true;
+    }
+    if (term == "pollo") {
+	return true;
+    }
+    return false;
+}
+
+
+export function getValue(term: string): number {
+    if (isChicken(term)) {
+	return 2000;
+    } else {
+	return 0;
+    }
```

Comment:
suggestion:

We could use a ternary here to be more concise:

```suggestion
    return isChicken(term) ? 2000 : 0;
```